From c8cd70209517afd6d781aa1884c492533c1203ab Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Mon, 5 Jan 2015 11:53:05 -0500
Subject: [redhat] kabi: Adding symbol pci_clear_master (drivers/pci/pci.c)

Message-id: <a2b043c2c2a74301f023f38c9ca751218ffee554.1420451770.git.poros@redhat.com>
Patchwork-id: 102441
O-Subject: [PATCH RHEL7.1 08/24] kabi: Adding symbol pci_clear_master (drivers/pci/pci.c)
Bugzilla: 1085963
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: 1085963
    https://bugzilla.redhat.com/show_bug.cgi?id=1085963

    Description:
    ============
    Used on shutdown to disable bus mastering. Side effect of calling pci_disable_device.
    Used in hpdsa.ko, hpvsa.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1070457 07/18] kabi: Adding symbol pci_clear_master (drivers/pci/pci.c)
 Jiri Olsa <jolsa@redhat.com> (Thu, 27 Feb 2014 10:03:21 +0100)
 Description:
 ============
 used by raidstack library to manipulate the pci bus

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pci_clear_master
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/pci_clear_master
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pci_clear_master
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pci_clear_master

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pci_clear_master b/redhat/kabi/kabi-module/kabi_ppc64/pci_clear_master
new file mode 100644
index 0000000..526af81
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pci_clear_master
@@ -0,0 +1,2 @@
+#1-
+0xa42ab0fe pci_clear_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/pci_clear_master b/redhat/kabi/kabi-module/kabi_ppc64le/pci_clear_master
new file mode 100644
index 0000000..826a1b5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/pci_clear_master
@@ -0,0 +1,2 @@
+#1-
+0x6a4bae3a pci_clear_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pci_clear_master b/redhat/kabi/kabi-module/kabi_s390x/pci_clear_master
new file mode 100644
index 0000000..aeb05a3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pci_clear_master
@@ -0,0 +1,2 @@
+#1-
+0x74d8ca94 pci_clear_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pci_clear_master b/redhat/kabi/kabi-module/kabi_x86_64/pci_clear_master
new file mode 100644
index 0000000..01ad46b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pci_clear_master
@@ -0,0 +1,2 @@
+#1-
+0x909b4067 pci_clear_master vmlinux EXPORT_SYMBOL
-- 
1.7.1